api/src/main/java/uk/gov/hmcts/fees2/register/api/controllers/FeeController.java

```diff
@@ -165,6 +165,10 @@ public void deleteFee(@PathVariable("code") String code) {
                                                                 HttpServletResponse response) {
         /* These are provisional hacks, in reality we need to lookup versions not fees so we require a massive refactor of search */
 
+        if (feeVersionStatus != null && feeVersionStatus.equals(FeeVersionStatus.approved)) {
```

Comment:
this one should just say feeVersionService.getDraftVersion.getFeesVersion(feeVersionStatus)